src/currency.js

```diff
@@ -146,6 +146,21 @@ currency.prototype = {
     return currency(intValue /= parse(number, _settings, false), _settings);
   },
 
+  /**
+   * Compares value to another currency instance's value
+   * @param {number|currency} foreign 
+   * @returns {boolean}
+   */
+  equals(foreign) {
```

Comment:
Drive-by comment: "foreign" has a specific connotation in dealing with currencies (e.g. "foreign exchange rate") which falsely implies that this may deal with currency conversions. Perhaps another variable name like `other` would be less ambiguous?